src/libs/kata-sys-util/src/mount.rs

```diff
@@ -227,7 +229,7 @@ pub fn bind_remount<P: AsRef<Path>>(dst: P, readonly: bool) -> Result<()> {
     do_rebind_mount(dst, readonly, MsFlags::empty())
 }
 
-/// Bind mount `src` to `dst` in slave mode, optionally in readonly mode if `readonly` is true.
+/// Bind mount `src` to `dst` with a custom propagation type, optionally in readonly mode if `readonly` is true.
 ///
```

Comment:
Thx.
Only two propagation types found. I think,  add a comment about types supported may be better.